diff --git a/CHANGELOG.md b/CHANGELOG.md
deleted file mode 100644
index dc8fceb28336c8d04ef38c0130242d06570e2eff..0000000000000000000000000000000000000000
diff --git a/rtf-interpreter.js b/rtf-interpreter.js
index 09a89dda3b4dc6208f42c4413ad5f0e578d2d064..137b6bb8fb0723f7eb76e8ae474fef55aebf2771 100644
--- a/rtf-interpreter.js
+++ b/rtf-interpreter.js
@@ -10,7 +10,8 @@ const iconv = require('iconv-lite')
 
 const availableCP = [
   437, 737, 775, 850, 852, 853, 855, 857, 858, 860, 861, 863, 865, 866,
-  869, 932, 936, 949, 950, 1125, 1250, 1251, 1252, 1253, 1254, 1257 ]
+  869, 932, 936, 949, 950, 1125, 1250, 1251, 1252, 1253, 1254, 1257,
+  10000 ]
 const codeToCP = {
   0: 'ASCII',
   2: 'SYMBOL',
@@ -241,6 +242,8 @@ class RTFInterpreter extends Writable {
   ctrl$ansicpg (codepage) {
     if (availableCP.indexOf(codepage) === -1) {
       this.emit('error', new Error('Codepage ' + codepage + ' is not available.'))
+    } else if (codepage === 10000) {
+      this.group.charset = 'MacRoman'
     } else {
       this.group.charset = 'CP' + codepage
     }
